### ISSUE TITLE
Deleting Prefix and Suffix

### ISSUE THREAD

**Issue by savoryninja1** [SkriptLang/Skript#5644]
### Skript/Server Version

```markdown
Papermc 1.19.3
Skript 2.6.4
SkBee 2.8.4
Latest Vault
Luckperms 5.4.62
```

### Bug Description

You should be able to delete the player's prefix or suffix if it was set, so that it can be delete  from the player's prefix so that it will default to nothing or to the player's ranks prefix

### Expected Behavior

You should be able to do `(delete|clear) player's (prefix|suffix)` current prefix or suffix if it was set with `set player's prefix "test"` so that the prefix/suffix will go away, so that the prefix/suffix on the player gets delete and defaults to nothing or the player's rank's prefix/suffix

### Steps to Reproduce

set player's group to a group that has a prefix or suffix
ex: `/lp creategroup test`, `/lp group test meta setprefix "&ctest"`, `set player's groups to "test"`

player now has group test with the prefix "&ctest"
now set the player's prefix to something new
ex: `set player's prefix to "&bnew"`

now you should be allowed to delete the prefix "&bnew" of the player and it should default to the player's ranks prefix "&ctest"
ex: `delete player's prefix`

Now the player's prefix should be "&ctest" but you cant delete prefix and suffixs you can only set them like
`set player's prefix to ""` but that doesn't help setting the player's prefix to their ranks prefix, because you cant grab a groups prefix or suffix in skript

### Other

This is more of a suggestion to add the deletion of prefix and suffix, but I put it as a bug because player should be able to remove the prefix or suffix, just like similarly when you remove prefix or suffixs using a permission plugin like luckperms

would be nice if this was fixed with https://github.com/SkriptLang/Skript/issues/1932 if not thats fine

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by GDudeUnderscore** [comment]
real

**Comment by Alphabastii** [comment]
yes this needed for script xD i can't remove my prefix from script i have normal a luckperms prefix

**Comment by annabelsandford** [comment]
I’m not sure how this is considered "Low Priority" when it should clearly be marked as high. The fact that Skript overwrites prefixes and doesn’t provide an option to clear them and revert to defaults—such as LuckPerms prefixes—is a server-breaking issue.

**Comment by Fusezion** [comment]
This I believe was completed in https://github.com/SkriptLang/Skript/pull/6970

**Comment by annabelsandford** [comment]
> This I believe was completed in #6970

Is this a prod feature? Tested on 2.9.5:
Expr: "`delete player's prefix`" yields:
`prefix of the player can't be deleted/cleared`

**Comment by Asleeepp** [comment]
> > This I believe was completed in #6970
> 
> Is this a prod feature? Tested on 2.9.5: Expr: "`delete player's prefix`" yields: `prefix of the player can't be deleted/cleared`

This is for a feature release, not patch, you can get this by downloading the 2.10 beta

**Comment by annabelsandford** [comment]
> > > This I believe was completed in #6970
> > 
> > 
> > Is this a prod feature? Tested on 2.9.5: Expr: "`delete player's prefix`" yields: `prefix of the player can't be deleted/cleared`
> 
> This is for a feature release, not patch, you can get this by downloading the 2.10 pre-release

Thanks so much, I really appreciate it!
